client/app/components/accounts/wizard/creation.cjsx

```diff
@@ -0,0 +1,186 @@
+        else if discoverReq.status is RequestStatus.ERROR and
+        checkReq.status is null
+            _.extend state,
+                alert: ALERTS.DISCOVER_FAILED
+                isDiscoverable: false
+
+
+        # returns state with its fallback default values.
+        #
+        # `isDiscoverable` is true by default since no check has been performed
+        # (else, it means a previous config - discovered or manual - was
+        # submitted and discovering must be manually set by closing advanced
+        # settings panel)
+        _.defaults state,
+            isBusy:         isBusy
+            isDiscoverable: checkReq.status is null
+            alert:          null
+
+
+    componentWillUpdate: (nextProps, nextState) ->
+        # Only enable submit when a request isnt performed in background and
+        # if required fields (email / password) are filled
+        nextState.enableSubmit = not nextState.isBusy and
+            not _.isEmpty(nextState.login) and
+            not _.isEmpty(nextState.password)
+
+
+    render: ->
+        <section className='settings'>
+            <h1>{t('account wizard creation')}</h1>
+
+            <Form ns="account-wizard-creation" className="content">
+                <Form.Input type="text"
+                            name="login"
+                            label={t('account wizard creation login label')}
+                            value={@state.login}
+                            onChange={_.partial @updateState, 'login'} />
+                <Form.Input type="password"
+                            name="password"
+                            label={t('account wizard creation password label')}
+                            value={@state.password}
+                            onChange={_.partial @updateState, 'password'} />
+
+                {<div className="alert">
+                    <p>{t("account wizard alert #{@state.alert}")}</p>
+                    {<p>
+                        {t("account wizard alert oauth")}
+                        <a href={OAuthDomains[@state.OAuth]} target="_blank">{t("account wizard alert oauth link label")}</a>.
+                    </p> if @state.OAuth}
+                </div> if @state.alert}
+
+                <Servers expanded={not @state.isDiscoverable}
+                         legend={t('account wizard creation advanced parameters')}
+                         onExpand={@onExpand}
+                         onChange={@updateState}
+                         {..._.omit @state, 'isOAuth', 'isDiscoverable', 'isBusy'} />
+
+                <footer>
+                    <nav>
+                        {<button type="submit"
+                                onClick={@create}
+                                disabled={not @state.enableSubmit}>
+                            {t('account wizard creation save')}
+                        </button> unless @state.success}
+                        {<a href="#" className="alert success">
```

Comment:
what is happening when `onClick`? It should have a `onClick` or real `URL` into `href`attribute.
It it isnt a link : use button?